tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/ResolvedJSDependency.scala

```diff
@@ -0,0 +1,23 @@
+package org.scalajs.core.tools.jsdep
+
+import org.scalajs.core.tools.io._
+
+/** A dependency on a native JavaScript library that has been successfully
+ *  resolved
+ */
+final class ResolvedJSDependency(
+    val lib: VirtualJSFile,
+    val minifiedLib: Option[VirtualJSFile],
+    val info: ResolutionInfo)
+
+object ResolvedJSDependency {
+  /** Absolute minimum for a [[ResolvedJSDependency]]:
+   *
+   *  - The library itself
+   *  - Its relative name (lib.name)
+   */
+  def minimal(lib: VirtualJSFile): ResolvedJSDependency = {
```

Comment:
Oh, you meant for the method? Hmmm... A `ResolvedJSDependency` is always "from a file", but carries additional metadata. I kind of want to make it explicit in the name that this is not the only correct way of constructing a `ResolvedJSDependency` from a file, but the simplest one "that works"...